bin/rails

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env ruby
-begin
```

Comment:
why did you delete these lines?

Reply:
I
![image](https://user-images.githubusercontent.com/38437488/63008354-20b56080-be8b-11e9-8eb9-9129cd60465e.png)